### PR TITLE
Add default setting values to docs.

### LINKED ISSUE #1533: [Docs] Settings defaults mismatched in source code and documentation
# 📚 Documentation/Examples
Separating out from #1526 

The defaults shown in the settings don't consistently match the source code.

### For example:  
settings.memory_efficient is [True in the documentation](https://docs.gpytorch.ai/en/stable/settings.html#gpytorch.settings.memory_efficient), but [False in the corresponding source code](https://docs.gpytorch.ai/en/stable/_modules/gpytorch/settings.html#memory_efficient).

### PR DESCRIPTION
**Pull request by gpleiss**
[Fixes #1533]

### CODE REVIEW

File: gpytorch/settings.py [hunk 1]
```diff
@@ -518,6 +557,8 @@ class memory_efficient(_feature_flag):
     Whether or not to use Toeplitz math with gridded data, grid inducing point modules
     Pros: memory efficient, faster on CPU
     Cons: slower on GPUs with < 10000 inducing points
+
+    (Default: True)
     """
 
     _default = False
```
Comment (Balandat) [inline gpytorch/settings.py:564]: `True == False`?

Comment (gpleiss) [reply gpytorch/settings.py:564]: we have bigger problems if it does...

File: gpytorch/settings.py [hunk 2]
```diff
@@ -518,6 +557,8 @@ class memory_efficient(_feature_flag):
     Whether or not to use Toeplitz math with gridded data, grid inducing point modules
     Pros: memory efficient, faster on CPU
     Cons: slower on GPUs with < 10000 inducing points
+
+    (Default: True)
```
Comment (gpleiss) [inline gpytorch/settings.py:561]:
```suggestion
    (Default: False)
```

File: gpytorch/settings.py (hunk 1 above)
Comment (Balandat) [reply gpytorch/settings.py]: `P=NP`?

**Pull request merged**